pygmt/src/colorbar.py

```diff
@@ -47,6 +47,12 @@ def colorbar(self, **kwargs):
     ----------
     frame : str or list
         Set color bar boundary frame, labels, and axes attributes.
+    annotation : int or float or str
+        Set the interval for annotated gridlines on the colorbar.
+    xlabel : str or int or float
+        Set the label for the x-axis of the colorbar.
+    ylabel : str or int or float
+        Set the label for the y-axis of the colorbar.
```

Comment:
Probably should just set the type as `str`. Yes they could pass in a number, but that's not very common.
```suggestion
    xlabel : str
        Set the label for the x-axis of the colorbar.
    ylabel : str
        Set the label for the y-axis of the colorbar.
```